dll/apisets/api-ms-win-core-sysinfo-l1-2-0.spec

```diff
@@ -17,7 +17,7 @@
 @ stdcall GetSystemTime() kernel32.GetSystemTime
 @ stdcall GetSystemTimeAdjustment() kernel32.GetSystemTimeAdjustment
 @ stdcall GetSystemTimeAsFileTime() kernel32.GetSystemTimeAsFileTime
-@ stub GetSystemTimePreciseAsFileTime
+@ stdcall -version=0x602+ GetSystemTimePreciseAsFileTime() kernel32.GetSystemTimePreciseAsFileTime
```

Comment:
Don't we have the "these files are auto-generated and shouldn't be touched by hand" problem again here?